.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,28 @@
+---
+    name: benchmark tests
```

Comment:
Should we consolidate or reconcile with the tcp-h workflow?

Reply:
The tpch workflow should be removed now. I'll remove it